.env.sample

```diff
@@ -1,7 +1,6 @@
 RPC_URL=
```

Comment:
Add `RPC_URL_HOLESKY`

Reply:
added a note, no need to change vars here